### PR TITLE
small screen styling

### PR DESCRIPTION
**Pull request by ghost**
addresses #101 
Adds small screen styling for home page

### CODE REVIEW

File: components/team.js [hunk 1]
```diff
@@ -4,18 +4,21 @@ import TeamMember from './teamMember'
 const teamMembers = [
   {
     name: 'Joel Wasserman',
+    title: 'Outreach Director',
```
Comment (joelwass) [inline components/team.js]: what are these, why do we need titles

Comment (korlando7) [reply components/team.js]: They were in the design

**Review by joelwass — CHANGES_REQUESTED** [review]
From width 480px (responsive chrome dev tools) to 600px 

and then from 770px to 950px the first section seems to be broken i.e. the title doesn't show. this might not be related to your changes

File: components/team.js (hunk 1 above)
Comment (joelwass) [reply components/team.js]: can we remove them, i think they were just a placeholder

File: pages/index.js [hunk 2]
```diff
@@ -92,9 +92,9 @@ class IndexPage extends Component {
         <section className='index__nav'>
           <Nav />
         </section>
-        <section className='index__header h-section pv6 pv7-ns ph2 pr7-l cover-l'>
-          <div className='fr flex flex-column mt6 mt0-l w-90-m'>
-            <div className='white f1-l tf-oswald fl ml-4-m f-5-m pt7-m'>
+        <section className='index__header h-section v7-ns pr7-l cover-l'>
```
Comment (joelwass) [inline pages/index.js]: can you revert lines 95-97 since i just modified them

Comment (korlando7) [reply pages/index.js]: reverted

File: components/team.js (hunk 1 above)
Comment (korlando7) [reply components/team.js]: removed

**Review by joelwass — APPROVED** [review]

**Pull request merged**